.buildkite/build.rayci.yml

```diff
@@ -160,3 +160,9 @@ steps:
     soft_fail: true
     depends_on:
       - forge
+
+  - label: ":book: LinkCheck"
```

Comment:
nit: now see this again, can me move it to lint.rayci.yml; not really a build; also maybe follow the existing name format from the same file, thankks

Reply:
Will do now :+1: